src/routes/spa/spa-router.ts

```diff
@@ -17,16 +17,19 @@ let indexHtmlContent: string = "";
 
 SpaRouter.use(JwtHandlerWithoutQsh);
 SpaRouter.use("/*", async (_, res) => {
+	const { jiraHost } = res.locals;
+	const featureFlags = {
+		ENABLE_5KU_BACKFILL_PAGE: await booleanFlag(BooleanFlags.ENABLE_5KU_BACKFILL_PAGE, jiraHost)
```

Comment:
The `jiraHost` is not available at this stage, as this is the design principle of SPA:
1. the static html/js/css is always static, no authentication token will be passed in the AUTHORIZATION header, hence the router won't have jiraHost in the res.locals

---------

Having say that, we can do either of bellow:

1. Read the query param (`jwt`) here directly, and try to decode it (by reusing some logic the JwtHandler) and get the jiraHost. But don't fail the page if token not availabe, as the defer install will need this page for GH admin (which dosen't have jira token)
2. Or alternatively, in the spa Connected component, always do `AP.navigator.go("addonmodule", { moduleKey: "gh-addon-admin" });` and now do that navigation in the backend routing.